CSC8503/LevelManager.cpp

```diff
@@ -318,7 +316,9 @@ void LevelManager::InitialiseAssets() {
 	mRigAnimationStand = mRenderer->LoadAnimation("Max/Idle.anm");
 	mRigAnimationWalk = mRenderer->LoadAnimation("Max/Walk2.anm");
 	mRigAnimationSprint = mRenderer->LoadAnimation("Max/Incentivise.anm");
-
+	//preLoadtexID   
+	mAnimation->PreloadMatTextures(*mRenderer, *mGuardMesh,*mGuardMaterial, mPlayerTextures);//I use Guard mesh to player
```

Comment:
Rather than adding comments inline, adding them on the top of the makes it more visible.
And please add your nicknames/names on the comment lines!
✌️ 

//(Chris): I use Guard mesh to player 
mAnimation->PreloadMatTextures(*mRenderer, *mGuardMesh,*mGuardMaterial, mPlayerTextures);